packages/cli/test/integration/generators/repository.integration.js

```diff
@@ -289,6 +289,29 @@ describe('lb4 repository', function() {
       );
     });
 
+    it('allows other connectors', async () => {
```

Comment:
maybe add some check to verify the custom connector is installed or get added into the package.json file?